deep_qa/models/multiple_choice_qa/tuple_inference.py

```diff
@@ -175,3 +177,99 @@ def _build_model(self):
         final_output = MaskedSoftmax()(options_probabilities)
 
         return DeepQaModel(input=[question_input, background_input], output=[final_output])
+
+    def debug_display_instance_tuple_match_scores(self, answer_tuples, background_tuples, tuple_matcher_output,
```

Comment:
Also, this is a nice descriptive name, but it's also a bit long.  Is there a shorter name that's just as good?  Maybe something like `_render_tuple_match_scores`?